fourmolu.yaml

```diff
@@ -1,5 +1,5 @@
 # # Number of spaces per indentation step
-# indentation: 4
+indentation: 2
```

Comment:
Note that this was 4 before but the entire codebase is formatted using 2, so I changed it to reflect the actual situation.